WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift

```diff
@@ -0,0 +1,221 @@
+import UIKit
+
+class CommentContentTableViewCell: UITableViewCell, NibReusable {
+
+    // determines the state of the like button.
+    enum LikeState {
+        case unliked
+        case liked
+    }
+
+    // all the available images to display for the accessory button.
+    enum AccessoryButtonType {
+        case share
+        case threeDots
+    }
+
+    // MARK: - Public Properties
+
+    var displayName: String? = "" {
+        didSet {
+            nameLabel?.setText(displayName ?? "")
+        }
+    }
+
+    var displayDate: String? = "" {
```

Comment:
It maybe be too early to nitpick these, but these empty strings confuse me. I'm not sure why they're defaulting to empty strings (instead of nil). But I also think once you populate them this will change anyway, so...

Reply:
Yeah, will update them later. The variable was initially `String`, but I figured later on it should just be an optional instead since `text` properties on the labels are optional strings too. ...But I forgot to update the default value for it. Thanks for noticing!

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17109/commits/05d5a31e08a13244e2a493f9cf2aea0b40f07f88. I removed the properties altogether since they're configured through a `Comment` object.